custom_container/custom_container_fc.yaml

```diff
@@ -0,0 +1,92 @@
+parameter_defaults:
+  DockerCinderVolumeImage: <undercloud_full_hostname>:8787/rhosp-rhel8/openstack-cinder-volume-hpe:1.0
```

Comment:
The tag should point to latest.

Refer https://github.com/hpe-storage/hpe-3par-cinder-rhosp13/blob/master/custom_container/custom_container_fc.yaml#L2

Once we push the container image to redhat registry, it will have the tag number and we can mark the tag number alongwith latest.

Reply:
Ok. Will change the tag to "latest".